### DISCUSSION TITLE
May 27 Quansight Show & Tell

### DISCUSSION THREAD

**Discussion by tonyfast** [Quansight/Quansight.github.io#27] [Show and tell]
May 27 show & tell notes. Please use this discussion to mark highlights and links from our discussions.

DO NOT DISCLOSE CONFIDENTIAL CUSTOMER INFORMATION

## [call to action] :+1: :-1:  attendance

react to this discussion so we can take attendance.

### Q1 how's your energy

* :+1: i feel energetic from this week
* :-1: i feel exhausted from this week

### back up plans

* tell us about your hobbies
* play with jupyter rtc https://gke.mybinder.org/v2/gh/jupyterlab/jupyterlab/570819a0f9bc51e344ee0a68aebe8c84573a0de8

**Comment by tonyfast** [comment]
an example binder with rtc working https://gke.mybinder.org/v2/gh/jupyterlab/jupyterlab/570819a0f9bc51e344ee0a68aebe8c84573a0de8

**Comment by isabela-pf** [comment]
### Timestamps
Tony/Jupyter RTC: ~16 minutes 
Thomas/Sparse Matrices 101: ~37 minutes
Free for all: ~51 minutes
- Installing JLab RTC ~52–56 minutes
- JupyterLite ~57–60 minutes

**Comment by tonyfast** [comment]
links to things we discussed today _not in any specific order_

> the markdown is written so it can be copied as a reference into your document.

[pyo3] : https://pyo3.rs/v0.12.4/
[maturin]: https://github.com/PyO3/maturin 
[jupyterlite]: https://github.com/jtpio/jupyterlite
[pyodide]: https://pyodide.org/en/stable/
[stdlib js]: https://stdlib.io/
[jupyterlab rtc release candidate]: https://github.com/jupyterlab/jupyterlab/releases/tag/v3.1.0a10
[binder configs]:  https://mybinder.readthedocs.io/en/latest/using/config_files.html
[jupyterlab binder configs]: https://github.com/jupyterlab/jupyterlab/tree/master/binder/
[using rtc release candidate configuration]: https://github.com/jupyterlab/jupyterlab/blob/master/binder/jupyter_notebook_config.py#L12
[sparse matrices]: https://en.wikipedia.org/wiki/Sparse_matrix
[dale tovar's sparse data post]: https://www.quansight.com/post/leveraging-sparsity-for-efficient-data-manipulation-the-gcxs-sparse-array-format
[`scipy` sparse]: https://docs.scipy.org/doc/scipy/reference/sparse.html
[pydata sparse]: https://sparse.pydata.org/en/stable/
[air speed velocity `asv` benchmarking]: https://github.com/airspeed-velocity/asv
